napari/_qt/widgets/qt_welcome.py

```diff
@@ -172,3 +174,11 @@ def set_welcome_visible(self, visible=True):
     def resizeEvent(self, event):
         self.resized.emit()
         return super().resizeEvent(event)
+
+    def enterEvent(self, event):
+        self.enter.emit()
+        super().enterEvent(event)
+
+    def leaveEvent(self, event):
+        self.leave.emit()
+        super().leaveEvent(event)
```

Comment:
vispy does not provide leave/enter events so there is a need to workaround this.